pkg/controller/controller.go

```diff
@@ -438,6 +438,12 @@ func (cont *AciController) Run(stopCh <-chan struct{}) {
 		panic(err)
 	}
 
+	// Build Pod/NodeBD Dn's if not defined.
+	if cont.config.AciPodBdDn == "" || cont.config.AciNodeBdDn == "" {
+		cont.config.AciPodBdDn = "uni/tn-" + cont.config.AciPolicyTenant + "/BD-" + cont.config.AciPolicyTenant + "-pod-bd"
+		cont.config.AciNodeBdDn = "uni/tn-" + cont.config.AciPolicyTenant + "/BD-" + cont.config.AciPolicyTenant + "-node-bd"
+	}
```

Comment:
Is there a reason we changed this from the way it was before? I believe we were setting to "" in the overlay case and hence wanted to skip any further validation in that case.

Reply:
Ok good point, haven't thought of Overlay case in that case we need to remove this and also introduce the following checks below cont.config.AciPodBdDn != "". and i hope ACC provision will always provides that config value in case of non overlay mode